AttitudeManager/FlightModes/Inc/AM_ControlAlgorithm.hpp

```diff
@@ -13,9 +13,9 @@
 
```

Comment:
file name needs to be lowercase I believe: https://uwarg-docs.atlassian.net/wiki/spaces/ZP/pages/2268692486/Style+Standards

Reply:
Should this be done as part of this PR? As well as AM.hpp -> am.hpp?